src/features/movies/MovieList/MovieBox/index.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import {
+  MoviesListWrapper,
+  Image,
+  MovieWrapper,
+  LinkToMoviePage,
+  RatingWrapper,
+  Star,
+  Rate,
+  Votes,
+  Title,
+  DescriptionWrapper,
+  GreySubtitle,
+  TagsWraper,
+  Tag,
+} from "./styled";
+
+const MovieBox = ({ genres, movies }) => (
+  <MoviesListWrapper>
+    {movies.map((movie) => (
+      
+        <LinkToMoviePage
+          key={movies.indexOf(movie)}
+          to={`/movie/:id${movie.id}`}
+        >
+          <MovieWrapper>
+            {movie.poster_path ? (
+              <Image
+                src={`https://image.tmdb.org/t/p/original/${movie.poster_path}`}
+                alt=""
+              />
+            ) : (
+              <Image />
+            )}
+            <DescriptionWrapper>
+              {movie.title && <Title>{movie.title}</Title>}
+              {movie.release_date && (movie.department || movie.charakter) ? (
```

Comment:
shouldn't it be movie.character (with C not K)?

Reply:
I just corrected.